notebooks/GraphNotebook.ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
Thank you for this notebook, I have some minor comment about it:

- As in the user-guide notebook, it would be nice to have a title at the beginning
- Change 'Networkit' to 'NetworKit'
- When using NetworKit in Python, I think that it would be more appropriate to write `networkit.graph` rather than `Networkit::Graph`. The same applies to all the other points in the notebook where you use `Graph::<something>`
- Also, when you write `networkit.graph` it would be nice if they also were links to the `networkit.graph` documentation (i.e., [`networkit.graph`](https://networkit.github.io/dev-docs/python_api/networkit.html?highlight=graph#networkit.Graph)). If it is not too much work, this could be done for all the method used in this notebook (e.g., [`graph.removeNode`](https://networkit.github.io/dev-docs/python_api/graph.html?highlight=removenode#networkit.graph.Graph.removeNode))
- Unnecessary space in `print (G.hasNode(2))`
- Maybe include a small example where you show how to use the `graph.indexEdges()` function and the resulting indexing of the edes

Reply:
I agree was the comments, but maybe a small correction: It's `networkit.Graph` (capitalized G).

Reply:
I have made and pushed the the suggested changes.

Reply:
Thank you!
I noticed some further minor changes to be made:
- Change`addEdges` -> `addEdge`.
- As @avdgrinten wrote, change `networkit.graph` -> `networkit.Graph`, sorry for the mistake in my suggestion.
- Change `indexEdges(bool force = false)` to `indexEdges(force = False)` (be careful not to mix C++ and Python syntax).
- The `forEdgesOf` link is wrong, change it to [`forEdgesOf`](https://networkit.github.io/dev-docs/python_api/networkit.html?highlight=foredgesof#networkit.Graph.forEdgesOf).